tensorflow/lite/tools/accuracy/ilsvrc/imagenet_model_evaluator.cc

```diff
@@ -144,6 +145,9 @@ class CompositeObserver : public ImagenetModelEvaluator::Observer {
       tflite::Flag::CreateFlag(kDelegateFlag, &params.delegate,
                                "Delegate to use for inference, if available. "
                                "Must be one of {'nnapi', 'gpu'}"),
+      tflite::Flag::CreateFlag(kNumRanksFlag, &params.num_ranks,
+                               "Generates the top-1 to top-k accuracy counts "
+                               "where k is number of rank. Default: 10"),
```

Comment:
`top-k accuracy values` since it returns fractions & `where k = num_ranks`.

Reply:
fixed.